tests/integration/generate_fixtures/common.py

```diff
@@ -0,0 +1,240 @@
+import contextlib
+import os
+import shutil
+import signal
+import socket
+import subprocess
+import tempfile
+import time
+
+from eth_utils import (
+    force_text,
+    is_checksum_address,
+)
+
+COINBASE = '0xdc544d1aa88ff8bbd2f2aec754b1f1e99e1812fd'
+COINBASE_PK = '0x58d23b55bc9cdce1f18c2500f40ff4ab7245df9a89505e9b1fa4851f623d241d'
+
+KEYFILE_DATA = '{"address":"dc544d1aa88ff8bbd2f2aec754b1f1e99e1812fd","crypto":{"cipher":"aes-128-ctr","ciphertext":"52e06bc9397ea9fa2f0dae8de2b3e8116e92a2ecca9ad5ff0061d1c449704e98","cipherparams":{"iv":"aa5d0a5370ef65395c1a6607af857124"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"9fdf0764eb3645ffc184e166537f6fe70516bf0e34dc7311dea21f100f0c9263"},"mac":"4e0b51f42b865c15c485f4faefdd1f01a38637e5247f8c75ffe6a8c0eba856f6"},"id":"5a6124e0-10f1-4c1c-ae3e-d903eacb740a","version":3}'  # noqa: E501
+
+KEYFILE_PW = 'web3py-test'
+KEYFILE_FILENAME = 'UTC--2017-08-24T19-42-47.517572178Z--dc544d1aa88ff8bbd2f2aec754b1f1e99e1812fd'  # noqa: E501
+
+RAW_TXN_ACCOUNT = '0x39EEed73fb1D3855E90Cbd42f348b3D7b340aAA6'
+
+UNLOCKABLE_PRIVATE_KEY = '0x392f63a79b1ff8774845f3fa69de4a13800a59e7083f5187f1558f0797ad0f01'
+UNLOCKABLE_ACCOUNT = '0x12efdc31b1a8fa1a1e756dfd8a1601055c971e13'
+UNLOCKABLE_ACCOUNT_PW = KEYFILE_PW
+
+GENESIS_DATA = {
+    "config": {
+        "chainId": 1337,
+        "homesteadBlock": 0,
+        "eip150Block": 0,
+        "eip155Block": 10,
+        "eip158Block": 10,
+        "eip160Block": 10
+    },
+    "nonce": "0x0000000000000042",
+    "alloc": {
+        COINBASE: {
+            "balance": "1000000000000000000000000000"
+        },
+        UNLOCKABLE_ACCOUNT: {
+            "balance": "1000000000000000000000000000"
+        },
+        RAW_TXN_ACCOUNT: {
+            "balance": "1000000000000000000000000000"
+        }
+    },
+    "timestamp": "0x00",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "extraData": "0x3535353535353535353535353535353535353535353535353535353535353535",
+    "gasLimit": "0x1000000",
+    "difficulty": "0x10000",
+    "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": COINBASE
+}
+
+
+def ensure_path_exists(dir_path):
+    """
+    Make sure that a path exists
+    """
+    if not os.path.exists(dir_path):
+        os.makedirs(dir_path)
```

Comment:
FYI, this entire utility is replacable by just using `os.makedirs(the_path, exist_ok=True)`